pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminTopicApiTest.java

```diff
@@ -71,6 +75,62 @@ protected void cleanup() throws Exception {
         super.internalCleanup();
     }
 
+    @Test
+    public void testDeleteNonExistTopic() throws Exception {
+        // Case 1: call delete for a partitioned topic.
+        final String topic1 = BrokerTestUtil.newUniqueName("persistent://public/default/tp");
+        admin.topics().createPartitionedTopic(topic1, 2);
+        admin.schemas().createSchemaAsync(topic1, Schema.STRING.getSchemaInfo());
+        Awaitility.await().untilAsserted(() -> {
+            assertEquals(admin.schemas().getAllSchemas(topic1).size(), 1);
+        });
+        try {
+            admin.topics().delete(topic1);
+            fail("expected a 409 error");
+        } catch (Exception ex) {
+            assertTrue(ex.getMessage().contains("please call delete-partitioned-topic instead"));
+        }
+        Awaitility.await().untilAsserted(() -> {
```

Comment:
Misses `pollDelay(2, TimeUnit.SECONDS)`?

Would you like to check if the schema exists?

Reply:
Fixed